apps/cli/src/commands/environment/create.env.ts

```diff
@@ -0,0 +1,76 @@
+import BaseCommand from '../base.command'
+import Logger from '../../util/logger'
+import {
+  CommandActionData,
+  CommandOption
+} from 'src/types/command/command.types'
+import { text } from '@clack/prompts'
+import { EnvironmentController } from '@keyshade/api-client'
+import {
+  CreateEnvironmentRequest,
+  CreateEnvironmentResponse
+} from '../../../../../packages/api-client/src/types/environment.types'
+export class CreateEnvironment extends BaseCommand {
+  getName(): string {
+    return 'create'
+  }
+
+  getDescription(): string {
+    return 'Create a new environment'
+  }
+
+  getOptions(): CommandOption[] {
+    return []
```

Comment:
You would need to specify the options in here before you can extract them in the `action` function